Reference/unittest_swift_fieldorder.pb.swift

```diff
@@ -429,10 +429,12 @@ struct Swift_Protobuf_OneofTraversalGeneration: SwiftProtobuf.Message, SwiftProt
       try visitor.visitSingularInt32Field(value: v, fieldNumber: 113)
     }
     try self.oConflictField?.traverse(visitor: &visitor, start: 126, end: 127)
+    try self.oConflictExtensionsStart?.traverse(visitor: &visitor, start: 201, end: 202)
     try visitor.visitExtensionFields(fields: _protobuf_extensionFieldValues, start: 202, end: 203)
-    try self.oConflictExtensionsStart?.traverse(visitor: &visitor, start: 201, end: 227)
```

Comment:
Note: with was a bug in the old code.  The generation trying to interleave oneofs and extensions wasn't complete.  Since the new code path uncovered the error in the old generation code; with the added benefit the new code is actually simplier/easier to follow.